integration-tests/utils/integration-helpers.js

```diff
@@ -26,22 +26,29 @@ const getTestWorkspaceName = () => `test-workspace-${Math.floor(Math.random() *
 
 const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
   const workspaceName = getTestWorkspaceName()
-  await page.evaluate((name, billingProject) => {
-    return window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
-  }, workspaceName, billingProject)
-
-  rawConsole.info(`created workspace: ${workspaceName}`)
+  try {
+    await page.evaluate((name, billingProject) => {
+      return window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
+    }, workspaceName, billingProject)
 
+    rawConsole.info(`created workspace: ${workspaceName}`)
+  } catch (e) {
+    throw Error(`Failed to create workspace: ${workspaceName}`)
```

Comment:
Would it be helpful to include `billingProject` in the error messages above and below?